sklearn/utils/validation.py

```diff
@@ -459,3 +467,40 @@ def check_symmetric(array, tol=1E-10, raise_warning=True,
             array = 0.5 * (array + array.T)
 
     return array
+
+
+def check_is_fitted(estimator, attributes,
+                    msg="This %(name)s instance is not fitted yet.", 
```

Comment:
Perhaps this could be more assistive for newbies, like adding "Call fit() before using."